.gitignore

```diff
@@ -21,3 +21,51 @@ de265-config.h
 *.ncb
 *.suo
 *.vcproj*
+
```

Comment:
Changes to `.gitignore` should be in a separate PR as they are unrelated to the new feature. Also you add a couple of entries that are already ignored.

Reply:
done

Reply:
https://github.com/strukturag/libde265/pull/202